src/openapiGenerator/openapiGenerator.ts

```diff
@@ -58,6 +60,12 @@ export function generatorFn() {
                     args.push("-p");
                     args.push(extraOptions.toString());
                 }
+                const outputPathIndex: number = args.findIndex((item: string) => item === GenerateOptionName.outputShort || item === GenerateOptionName.output) + 1;
```

Comment:
why do you add `+1` at the end?

Reply:
to get the actual variable index of the option

Reply:
But you are executing "findIndex" to get the `outputPathIndex` not the `nextOutputPathIndex`. It's a bit confusing. You should have the `findIndex` returns exactly the index you need, othewise, the logic inside might not be accurate.

Reply:
the option is actually acting as the indicator to find actual argument. I created a new function to get the actual argument index